blocks/gallery-block/index.story.jsx

```diff
@@ -99,3 +101,26 @@ export const withPassedInContent = () => {
     />
   );
 };
+
+export const withHiddenContentOptions = () => {
```

Comment:
Shouldn't this story have the values set to true so we can have visual tests showing the items not displayed?

Reply:
updated